web/src/app/shared/filter/filter.component.html

```diff
@@ -64,15 +64,25 @@
         <span class="material-icons md-20 blue">clear</span>
       </button>
     </div>
-    <div class="row">
-      <div class="col-xs-4 col-xs-offset-2">
-        <input type="text" [(ngModel)]="filter.filterParams.state" (keyup.enter)="search()" class="form-control" id="display-owner-state"
-          placeholder="Filter status">
+    <div class="row" style="margin-top: 1px;">
```

Comment:
DO NOT USE INLINE STYLING

Reply:
I remove all my inline styling. I change to selector style